test/RPM-GPG-KEY-CentOS-7

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
we shouldn't be pushing keys even if they are public @eranha

Reply:
Talked to @sashaCher is ok with that, these are public keys, the alternative is to download it online, if a URL changes it would break the build